Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -122,7 +135,8 @@ namespace EMStudio
             {
                 cameras.AddCamera(m_rotateCamera);
                 cameras.AddCamera(m_translateCamera);
-                cameras.AddCamera(m_orbitDollyScrollCamera);
+                cameras.AddCamera(m_dollyScrollCamera);
```

Comment:
update: Remove this and add to `m_orbitCamera->m_orbitCameras` (see above comment)